core/src/test/java/org/elasticsearch/search/fetch/subphase/highlight/HighlighterSearchIT.java

```diff
@@ -2435,6 +2435,7 @@ public void testPostingsHighlighterCommonTermsQuery() throws IOException {
 
     private static XContentBuilder type1PostingsffsetsMapping() throws IOException {
         return XContentFactory.jsonBuilder().startObject().startObject("type1")
+                .startObject("_all").field("enabled", true).endObject()
                 .startObject("properties")
                 .startObject("field1").field("type", "text").field("index_options", "offsets").endObject()
                 .startObject("field2").field("type", "text").field("index_options", "offsets").endObject()
```

Comment:
`testPostingsHighlighter` has a test that queries on the `_all` field and highlights other fields. Without this, it would fail.

I can remove the particular clause if you think that would be a better solution?